test/integration/elm/elm.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Parcel should probably be responsible for reading elm configs.

Or at least look for the configs so it can add it for invalidating cache.

Reply:
That makes sense, I've updated the asset so that it will attempt to locate the `elm.json` file and add it as a dependency if found.